app/Main.hs

```diff
@@ -1,21 +1,25 @@
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE OverloadedStrings   #-}
```

Comment:
This should probably turn into docs, and be removed from the project proper (see below comment)